modules/gumgumBidAdapter.js

```diff
@@ -318,6 +325,7 @@ function buildRequests (validBidRequests, bidderRequest) {
       sizes,
       url: BID_ENDPOINT,
       method: 'GET',
+      gpid: gpid,
```

Comment:
unless you put something in data., as you have data.pi above, or data.fp, it doesn't appear to flow through to your endpoint